package.json

```diff
@@ -1,18 +1,24 @@
 {
   "name": "nexus-fusion",
   "description": "The interface of Blue Brain Nexus, the open-source knowledge graph for data-driven science.",
-  "version": "1.9.1",
+  "version": "1.10.x",
```

Comment:
Shouldn't we avoid defining any version number at all since we have to do it manually and might forget? After all, it's not a package deployed on NPM, so the version number is not crucial.

Reply:
yeah you are right, I ll remove it